.github/workflows/version.yml

```
@@ -26,7 +26,7 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-name: Build
+name: Version
```

```suggestion
name: API Version
```
Let's use `API Version` for now.